cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database")
+        connect = None
+        while not connect:
+            try:
+                connect = connections["default"]
+                connect.cursor()
+            except OperationalError:
+                time.sleep(1)
```

Comment:
It is a good practice to log the exception information. This can be useful for debugging if the database connection fails.